doc/developers/advanced_installation.rst

```diff
@@ -64,11 +64,11 @@ feature, code or documentation improvement).
 
    If you installed Python with conda, we recommend to create a dedicated
    `conda environment`_ with all the build dependencies of scikit-learn
-   (namely NumPy_, SciPy_, and Cython_):
+   (namely NumPy_, SciPy_, Cython_ and Meson_):
```

Comment:
Shouldn't we rather reference and link to meson-python instead? It's easy to find the meson doc from the meson-python page but not necessarily the opposite.

Reply:
On top of this, I think the two references that were removed are more likely to be useful than Meson or meson-python docs:

- `pandas setup doc
  <https://pandas.pydata.org/docs/development/contributing_environment.html#step-3-build-and-install-pandas>`_:
  pandas has a similar setup as ours (no spin or dev.py)
- `scipy Meson doc
  <https://scipy.github.io/devdocs/building/understanding_meson.html>`_ gives
  more background about how Meson works behind the scenes

Reply:
I switched to linking to meson-python.

Where would be a good place to add the links Loic? In this bit of the docs we link to the various dependencies and their homepages, so it feels weird to add the two links which are more about background reading.

Reply:
Good question, maybe a note at the end of the "Building from source" section, maybe this should more go in `developers/tips.rst` since this is more suited for scikit-learn maintainers?

Reply:
I added a section to the tips page.